docs/api/page.rst

```diff
@@ -1095,7 +1096,17 @@ In a nutshell, this is what you can do with MuPDF.NET:
 
       This method mainly exists for technical purposes. Typical uses include replacing large images by smaller versions, like a lower resolution, graylevel instead of colored, etc., or changing transparency.
 
-   
+   .. index::
+      pair: pageNum; Recolor
+      pair: colorSpaceName; Recolor 
+
+   .. method:: Recolor(int colorNum)
+   .. method:: Recolor(string colorSpaceName)
```

Comment:
Please put in a carriage return here (see previous comment on Document.rst as to why )